lighthouse-core/config/budget.js

```diff
@@ -87,43 +117,44 @@ class Budget {
   /**
    * More info on the Budget format:
    * https://github.com/GoogleChrome/lighthouse/issues/6053#issuecomment-428385930
-   * @param {Array<LH.Budget>} budgetArr
+   * @param {unknown} budgetJson
```

Comment:
it seems a little weird to act like `cliFlags.budgets` is an `Array<LH.Budget>` in all of `lighthouse-cli/`, but then treat it distrustfully as soon as it gets into this file, but switching `SharedFlagsSettings['budgets']` to `unknown also seemed a bit silly because by that argument *all* of the CLI flags should be treated as `unknown` until we verify them.

And maybe we should be, but then that degrades the handiness and safety of having a single verification point inside `config.js`, and I'm not sure it's worth verifying in `bin.js` *and* in `config.js`...